packages/components/src/TransactionButton.tsx

```diff
@@ -129,6 +129,7 @@ export class TransactionButtonNoModal extends React.Component<TransactionButtonP
         return this.executeTransactions(transactions);
       } catch (err) {
         if (currTransaction.handleTransactionError) {
+          this.setState({ step: 0, disableButton: false });
```

Comment:
Should this be outside the `if` block? That way the buttons resets even if the transaction doesn't have a `handleTransactionError`.

Though I think all transactions buttons w/ the modal have `handleTransactionError` set, but lower level implementations of `TransactionButton` might not have that set.

Reply:
good point, i'll take it out of the if